.github/workflows/wheels.yaml

```diff
@@ -24,11 +24,11 @@ jobs:
             arch: x86
             cmake_generator: "Visual Studio 16 2019"
             cmake_generator_platform: "Win32"
-          - os: ubuntu-20.04
+          - os: ubuntu-latest
             arch: x86_64
-          - os: ubuntu-20.04
+          - os: ubuntu-24.04-arm
```

Comment:
```
The only
exception is the `aarch64` build on Linux that uses `ubuntu-24.04-arm`
which an `aarch64` runner which allows to build the wheels without
emulation speeding up the build process
```

So either
1) add to commit message information that removing qemu will be done later, or
2) remove QEMU now, or
3) just simply switch to ubuntu-latest and lets deal with it all later.

Note that ubuntu-arm images are STILL in preview, so updating to it is questionable without additional reasons.

-----
_off-topic_
Apparently I did that for segyio just 4 months ago and I had **no** memory of it at all  :sweat_smile: